src/protocols/core/Shm.cpp

```diff
@@ -99,10 +100,25 @@ void CSHMPool::resize(size_t size_) {
         LOGM(ERR, "Couldn't mmap {} bytes from fd {} of shm client", size, fd);
 }
 
+int shmIsSizeValid(int fd, size_t size) {
```

Comment:
this is fine, just make it `static` please

Reply:
Oh yes of course